src/__test__/components.test.js

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Home from '../components/pages/Home/Home';
+import Quote from '../components/pages/Quote/Quote';
+
+describe('Test different components', () => {
+  test('It should render Home', () => {
+    render(<Home />);
+    const author = screen.getByTestId('welcome');
+    expect(author).toHaveTextContent('Welcome to our page');
+  });
+});
+
+describe('Test different components', () => {
+  test('It should render Quote component', () => {
+    render(<Quote />);
+    expect(screen.getByText(/William Paul Thurston/)).toBeInTheDocument();
+  });
+});
```

Comment:
- [ ] @edenlisk and @collins-kiprotich great work on your components test, however, you need to add tests for all components , didn't see tests for the app navbar and calculator components, this is a project requirement
![image](https://user-images.githubusercontent.com/61408860/214594664-0400a57c-8a2a-43b8-90a9-4b7ca241eaf2.png)